modules/custom/moj_resources/src/Utilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\moj_resources;
+
+/**
+ * Utilities
+ */
+
+class Utilities
+{
+  public static function filterByPrisonCategories($prison_categories, $query) {
+    $prison_categories_or_group = $query
+      ->orConditionGroup()
+      ->condition('field_prison_categories', $prison_categories, 'IN')
+      ->condition('field_prison_categories', NULL, 'IS NULL') // this condition needs to remain in until we do a full launch otherwise no series will work
```

Comment:
Added a comment but if we commit this as intended, i.e. this and the one below removed then Series as it stands won't work, either that or we go through Series and add this data as a test?